src/headline_grabber/validators/click/option_validator.py

```diff
@@ -57,4 +57,14 @@ def validate_filter_sentiment(ctx, param, value):
         value = 'POSITIVE' if 'POS' in value else 'NEGATIVE'
 
         return value
+    
+    @staticmethod
+    def validate_keywords(ctx, param, value):
```

Comment:
Add a unit test or two for this function.

Reply:
Hi @mralderman, I will add the unit test for this option_validator.py class in the US next sprint.